src/templates/typescript/interfaces/interfaceTemplate.ts

```diff
@@ -0,0 +1,28 @@
+import IDataType from "../../../model/dataTypes/IDataType";
+import MemberVariable from "../../../model/dataTypes/MemberVariable";
+import Handlebars from "../registerPartials";
+
+export const source = `{{> interfaceImport dataType=this }}
+export default interface I{{ name }}{{# if baseDataType }} extends I{{ baseDataType.typeName }}{{/ if }} {
+{{# if memberVariables }}
+  {{# each memberVariables }}
+  {{# if dataType.primitive }}
+  {{> primitiveMember member=this }}
+  
+  {{ else }}
+  {{> interfaceMember member=this }}
+  
+  {{/ if }}
+  {{/ each }}
+{{/ if }}
+}
+`;
+
+export interface ITemplateContext {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5648116)